lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -180,4 +201,26 @@ module.exports = {
     }
     return event.alb.multiValueHeaders;
   },
+
+  validateEventAuthorizers(event, authorizers, functionName) {
+    const eventAuthorizers = Array.isArray(event.alb.authorizer)
+      ? event.alb.authorizer
+      : [event.alb.authorizer];
+    for (const auth of eventAuthorizers) {
+      if (!authorizers[auth]) {
+        throw new this.serverless.classes.Error(
+          `No match for "${auth}" in function "${functionName}" found in registered ALB authorizers`
+        );
+      }
+    }
+    return eventAuthorizers;
+  },
+
+  validateAlbAuth(auth) {
+    auth.onUnauthenticatedRequest = auth.onUnauthenticatedRequest = auth.allowUnauthenticated
```

Comment:
Assignment seems duplicated (wierd lint didn't pick this up)

Reply:
Don't know how I didn't pick it up 😅